diff --git a/source/vst/utility/alignedalloc.h b/source/vst/utility/alignedalloc.h
index 0ce85e7..4ca7556 100644
--- a/source/vst/utility/alignedalloc.h
+++ b/source/vst/utility/alignedalloc.h
@@ -25,7 +25,7 @@
 #include <AvailabilityMacros.h>
 #endif
 
-#ifdef _MSC_VER
+#if defined (_MSC_VER) || defined(__MINGW32__)
 #include <malloc.h>
 #endif
 
@@ -53,7 +53,7 @@ inline void* aligned_alloc (size_t numBytes, uint32_t alignment)
 #if SMTG_OS_MACOS && defined(MAC_OS_X_VERSION_MIN_REQUIRED) && \
     MAC_OS_X_VERSION_MIN_REQUIRED < MAC_OS_X_VERSION_10_15
 	posix_memalign (&data, alignment, numBytes);
-#elif defined(_MSC_VER)
+#elif defined(_MSC_VER) || defined(__MINGW32__)
 	data = _aligned_malloc (numBytes, alignment);
 #else
 	data = std::aligned_alloc (alignment, numBytes);
@@ -68,7 +68,7 @@ inline void aligned_free (void* addr, uint32_t alignment)
 		std::free (addr);
 	else
 	{
-#if defined(_MSC_VER)
+#if defined(_MSC_VER) || defined(__MINGW32__)
 		_aligned_free (addr);
 #else
 		std::free (addr);
